### PR TITLE
Kbattula/1331739

### PR DESCRIPTION
**Pull request by karthikhhr**
Here is the fix for the bug 1331739 [here](https://bugzilla.redhat.com/show_bug.cgi?id=1331739). Please take a look.

@cnsnyder @barnabycourt

### CODE REVIEW

**Comment by karthikhhr** [comment]
@cnsnyder  just in case

File: src/subscription_manager/managercli.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 # Red Hat trademarks are not licensed under GPLv2. No permission is
 # granted to use or replicate Red Hat trademarks that are incorporated
 # in this software or its documentation.
-#
 
+#
```
Comment (cnsnyder) [inline src/subscription_manager/managercli.py]: Why?

File: src/subscription_manager/managercli.py [hunk 2]
```diff
@@ -2597,7 +2597,14 @@ def _do_command(self):
 
         if self.options.additions:
             repo_ids = [repo.id for repo in overrides.repo_lib.get_repos(apply_overrides=False)]
-            to_add = [Override(repo, name, value) for repo in self.options.repos for name, value in self.options.additions.items()]
+            to_add = []
+            for name, value in self.options.additions.items():
```
Comment (barnabycourt) [inline src/subscription_manager/managercli.py]: iteritems is more efficient as it doesn't need to convert the dict to a list in memory

**Comment by karthikhhr** [comment]
@cnsnyder Please take a look now

**Comment by cnsnyder** [comment]
@karthikhhr Please rebase this on master to fix the issues with the jenkins-stylish test. Thanks!

**Comment by cnsnyder** [comment]
Fixed a few stylish issues. Waiting for jenkins to be back up and running before merging.

**Comment by cnsnyder** [comment]
@candlepin/jenkins-bot please retest this

**Comment by cnsnyder** [comment]
Closing in favor of #1474

**Pull request closed without merging**